Underanalyzer/Decompiler/GlobalFunctions.cs

```diff
@@ -60,29 +72,29 @@ public GlobalFunctions(IEnumerable<IGMCode> globalScripts, ParallelOptions paral
             List<Fragment> fragments = Fragment.FindFragments(script, blocks);
 
             // Find names of functions after each fragment
-            for (int i = 1; i < fragments.Count; i++)
+            foreach (Fragment fragment in fragments)
```

Comment:
The reason this was a for loop starting at index 1 was because the first fragment is always the root-level fragment, which is never a fragment belonging to a sub-function. However... the first if statement that checks for it being "corrupt" will immediately skip that fragment, so it's not a huge issue (unless any warnings were added, in which case it would be an issue).

Reply:
oops, i didnt even see it started at one.